.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on: pull_request
+
+jobs:
+  checkout:
+    name: Clone the repo
```

Comment:
I did this in parallel in the hopes that GitHub could be downloading the Bazel container in parallel but in retrospect that obviously isn't how actions jobs work I'll merge these jobs for simplicity.